src/screens/EventsScreen/index.js

```diff
@@ -18,19 +18,14 @@ type OwnProps = {
   navigation: NavigationScreenProp<NavigationState>
 };
 
-type Props = {
-  events: EventDays,
-  loading: boolean,
-  refreshing: boolean,
-  updateEvents: () => Promise<void>,
-  getAssetUrl: LocalizedFieldRef => string
-} & OwnProps;
+type Props = ComponentProps & OwnProps;
```

Comment:
These props where being duplicated from the component props. @daralthus came with this to reuse them.